test/docfx.Test/lib/ProcessUtilityTest.cs

```diff
@@ -65,22 +65,34 @@ public static async Task FileMutexTest()
         [Fact]
         public static async Task NestedRunInMutexTest()
         {
-            // works for one lock one file
-            await ProcessUtility.RunInsideMutex($"process-test/{Guid.NewGuid()}", async () => { await Task.Delay(100); });
-            await ProcessUtility.RunInsideMutex($"process-test/{Guid.NewGuid()}", async () => { await Task.Delay(100); });
-
-            // doesn't work for requiring a lock before releasing a lock
-            await Assert.ThrowsAnyAsync<Exception>(async () =>
-            {
-                await ProcessUtility.RunInsideMutex($"process-test/{Guid.NewGuid()}",
+            // nested run works for different names
```

Comment:
shall we split this into 2 test cases, it would be easier to detect which one is failing

Reply:
great idea